drivers/interrupt_controller/intc_gpio_stm32.c

```diff
@@ -102,7 +102,7 @@ static void stm32_intc_gpio_isr(const void *exti_range)
 	uint32_t line_num;
 
 	/* see which bits are set */
-	for (uint8_t i = 0; i <= range->len; i++) {
+	for (uint8_t i = 0; i < range->len; i++) {
```

Comment:
Nice catch!
Looks like a issue. Worth to report into v3.7 LTS?

Reply:
Oups, it should be in this PR, I made a dedicated one.
I'll have a look but IIRC this was merged after v4.2 release, so no need to backport.

Reply:
No, the issue is present in *intc_exti_stm32.c* before it was renamed to *intc_gpio_stm32.c* in v4.2. Looking into the source revision history, the issue is present since commit d7631a60eca8604fb719b2038de7258f16947c89, merged in v3.3. Unless I missed something.

Reply:
Exact, my bad (and my bug actually).
This being said, I'm not sure it's worth the pain to back-port it. Generally speaking I think we should globally minimize back porting to impacting issues.